include/boost/optional/detail/optional_utility.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2024 Andrzej Krzemienski.
+// Copyright (C) 2024 typenameTea.
```

Comment:
You prefer to remain anonymous? Ok.

Reply:
My quick research on the Internet indicates that a copyright notice is valid even when your pseudonym is used.
https://copyrightalliance.org/faqs/can-you-register-copyright-under-pseudonym/
https://copyrightservice.co.uk/copyright/p03_copyright_notices
Of course, I am not a lawyer, and stuff found on the Internet is not legally binding, but I consider it good enough.
It is just a new thing for me.

Reply:
In practical terms, it doesn't matter. Use what you like. For more information see:

https://www.copyright.gov/circs/circ32.pdf